ui/src/components/user.tsx

```diff
@@ -144,7 +118,7 @@ export class User extends Component<any, UserState> {
   constructor(props: any, context: any) {
     super(props, context);
 
-    this.state = this.emptyState;
+    this.state = Object.assign({}, this.emptyState);
```

Comment:
We haven't needed to use this in any other state setup, why is it necessary here?

Reply:
For both this and the below comment, I started working on this and experimenting a week ago and forgot to undo it when making the first commit 🤦 , reverted the changes